src/premium/awaitedReactions.md

```diff
@@ -1,26 +1,50 @@
 # Awaited reactions
 *(for premium bots)*
 
-Thanks to awaited reactions your bot can be triggered by reactions
+**Awaited reactions** are same like [awaited commands](../guides/awaitedCommands.md). Unlike awaited commands which waits for a message, it waits for a reaction instead. 
 
-![end result](https://i.imgur.com/diskuyv.png) \
+![end result](https://i.imgur.com/diskuyv.png)
 
-Here's how the end result looks like.
+> 📝 Reaction roles are not possible at the moment since awaited reactions can only be triggered by author and it gets expired whenever bot goes offline.
```

Comment:
Should be `by the author and they expire whenever the bot goes offline`

```suggestion
> 📝 Reaction roles are not possible at the moment since awaited reactions can only be triggered by the author and they expire whenever the bot goes offline.
```